src/Data/Swagger/Declare.hs

```diff
@@ -162,10 +162,6 @@ instance MonadDeclare d m => MonadDeclare d (IdentityT m) where
   declare = lift . declare
   look = lift look
 
-instance MonadDeclare d m => MonadDeclare d (ListT m) where
```

Comment:
Why remove this instance?

Reply:
it created a deprecation warning:

> Deprecated: This transformer is invalid on most monads

https://hoogle.zinfra.io/file/usr/share/doc/ghc-8.4.3/html/libraries/transformers-0.5.5.0/Control-Monad-Trans-List.html

I figured anybody who wants to use it can still re-create the instance, but they will be prompted to think about it at least.

I'm happy to revert that change if you disagree?

Reply:
"re-create the instance" would mean an orphan one. No go.

Reply:
but providing an instance in this library for a transformer that will almost always lead to broken monads is ok?  i disagree, but ok: cf0c2a770ae8c0200b3